.changelog/unreleased/bug-fixes/206-fix-historical-tribute-claims.md

```diff
@@ -0,0 +1,2 @@
+- Disable voting in next round with locks that were used in previous round to vote for longer deployment proposal through the "auto-voting" mechanism.
```

Comment:
c/p from a previous PR? :)

Reply:
oh I didn't push the change for this one!